api/src/main/resources/workbench.yaml

```diff
@@ -36,6 +36,12 @@ paths:
             type: "array"
             items:
               $ref: "#/definitions/Cohort"
+      security:
+        - aou_oauth: []
+securityDefinitions:
+  aou_oauth:
+    type: oauth2
```

Comment:
Added — note that I'm not sure whether/how the autogenerated server treats these. (I _assume_ not at all.)